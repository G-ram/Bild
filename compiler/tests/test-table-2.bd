a = {};
a[0] = {};
a[0][0] = "hey";
print(a);
