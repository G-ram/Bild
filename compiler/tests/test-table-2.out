0 : 0 : hey
